internal/tool/deploy.go

```diff
@@ -310,22 +309,20 @@ func (d *DeploySpec) startRollout(ctx context.Context, cfg *config.GKEConfig) er
 	return fmt.Errorf("timeout trying to deploy the app; last error: %w", err)
 }
 
-// pickDeployRegions ensures that the application config has a valid set of
+// validateDeployRegions ensures that the application config has a valid set of
 // unique regions to deploy the application. If the app config doesn't specify
 // any regions where to deploy the app, we pick the regions.
 //
-// TODO(rgrandl): We pick "us-west1" as the default region. However, we should
-// determine the set of regions to deploy the app based on various constraints
-// (e.g., traffic patterns, geographical location, etc.).
-func pickDeployRegions(cfg *config.GKEConfig) error {
-	if len(cfg.Regions) == 0 {
-		cfg.Regions = []string{"us-west1"}
-		return nil
+// Note that at least one regin should be specified.
+func validateDeployRegions(regions []string) error {
+	// Ensure that at least one region where to deploy the app is specified.
```

Comment:
`//Ensure that at lease one deployment region is specified.`

Reply:
Done